layouts/shortcodes/blocks/section.html

```diff
@@ -1,6 +1,6 @@
 {{ $col_id := .Get "color" | default .Ordinal }}
 {{ $height := .Get "height" | default "auto"  }}
-{{ $type   := .Get "type" | default "" }} 
+{{ $type   := .Get "type" | default "features" }} 
```

Comment:
While these all cause the following error:
```
hugo server
Building sites … ERROR 2019/06/18 10:15:08 error: failed to transform resource: SCSS processing failed: file 
"/usr/local/google/home/rescarez/docsytests/docsyexampletemplatetest/themes/docsy/assets/scss/blocks/_blocks.scss", 
line 1, col 1: Invalid CSS after "": expected 1 selector or at-rule, was '{{ $type := .Get "t' 
Total in 335 ms
Error: Error building site: logged 1 error(s)
```

Reply:
Huh, odd. Let me check with my local version, I think the Docsy example site is still running with an older Hugo/Docsy. If it works without it in both then I'll remove it.

Reply:
You're right, I didn't actually make it properly backwards compatible! Done.